openapi/components/schemas/QuoteCreateOrder.yaml

```
@@ -95,124 +77,43 @@ properties:
-                This field is useful for accounting and customer success purposes.
-                The usage of metered billing plans can still be reported if the soft limit is reached.
-              minProperties: 1
-              maxProperties: 1
-              properties:
-                quantity:
-                  type: integer
-                  description: Usage limit quantity.
-                  minimum: 1
-                amount:
-                  type: number
-                  format: double
-                  description: Usage limit amount in the currency of the order.
-                  minimum: 0.01
-            hardLimit:
-              type: object
-              description: |-
-                Defines a limit where the customer can no longer use the service.
-                Hard limits are used in addition to soft limits.
-                When a soft limit is reached,
-                a customer may receive a notification
-                but the service can still be provided up to the hard limit value so that the customer can upgrade their plan.
-                When the reported usage reaches the configured limit,
-                a specific event is triggered.
-                To notify the customer in the merchant system,
-                or block a service,
-                a webhook and notification can be configured for this event.
-                When the total usage reaches the hard limit quantity,
-                or amount values,
-                metered billing plan usages can no longer be reported.
-              minProperties: 1
-              maxProperties: 1
-              properties:
-                quantity:
-                  type: integer
-                  description: Usage limit quantity.
-                  minimum: 1
-                amount:
-                  type: number
-                  format: double
-                  description: Usage limit amount in the currency of the order.
-                  minimum: 0.01
-        createdTime:
-          $ref: ./CreatedTime.yaml
-        updatedTime:
-          $ref: ./UpdatedTime.yaml
-        _embedded:
-          type: object
-          description: Embedded objects that are requested by the `expand` query parameter.
-          readOnly: true
-          properties:
-            product:
-              type: object
-  deliveryAddress:
-    description: Delivery address of the order.
-    oneOf:
-      - $ref: ./ContactObject.yaml
-      - type: 'null'
-  billingAddress:
-    description: Billing address of the order.
-    oneOf:
-      - $ref: ./ContactObject.yaml
-      - type: 'null'
+  order:
+    type: object
+    description: Related order properties.
+    properties:
+      id:
+        description: ID of the related order.
+        readOnly: true
+        type:
+          - 'string'
+          - 'null'
+        maxLength: 50
+        example: ord_0YV7DES3WPC5J8JD8QTVNZBZNZ
```

It will be nice and consistent to have the same identifiers for each resource.

```suggestion
        example: ord_01HVKA5975PJBSQ1SX72G3MSZC
```

---

I think there are a couple of more places we should change if we want it to be consistent